practice-one/src/sections/service.html

```diff
@@ -1,34 +1,28 @@
 <section id="service-section" class="service-container">
   <div class="illustration-service-image"><img src="./assets/images/service.png" alt="Service's illustration"/></div>
   <div class="service-description-container">
-    <p class="service-header">We are <span class="primary-color">more</span> than <span class="accent-color">multiple</span> service</p>
+    <h2 class="service-header">We are <span class="primary-color">more</span> than <span class="accent-color">multiple</span> service</h2>
     <p class="service-description">This is a type of resturent which typically serves food and drink, in addition to light refreshments such as baked goods or snacks. The term comes frome the rench word meaning food.</p>
-    <div class="service-list-wrapper">
-      <div class="service">
+    <ul class="service-list-wrapper">
+      <li class="service">
         <div class="service-icon"><img src="./assets/images/order-online.svg" alt="Online order icon"/></div>
-        <p>Online order</p>
-      </div>
-      <div class="service">
+      </li>
+      <li class="service">
         <div class="service-icon"><img src="./assets/images/247service.svg" alt="Service icon"/></div>
-        <p>24/7 Service</p>
-      </div>
-      <div class="service">
+      </li>
+      <li class="service">
         <div class="service-icon"><img src="./assets/images/service-cake.svg" alt="Pre-Reservation icon"/></div>
-        <p>Pre-Reservation</p>
-      </div>
-      <div class="service">
+      </li>
+      <li class="service">
         <div class="service-icon"><img src="./assets/images/service-cake.svg" alt="super Chef icon"/></div>
-        <p>Super Chef</p>
-      </div>
-      <div class="service">
+      </li>
+      <li class="service">
         <div class="service-icon"><img src="./assets/images/service-cake.svg" alt="Organized Foodhut Place icon"/></div>
-        <p>Organized Foodhut Place</p>
-      </div>
-      <div class="service">
+      </li>
+      <li class="service">
```

Comment:
Class name is not good at all. However because you use it from your previous MR, so it's ok for now to by pass

Reply:
thanks

Reply:
I will pay more attention